### PR TITLE
css-layout-classes created - William Zhang

### PR DESCRIPTION
**Pull request by williamzhang73**
layout

### CODE REVIEW

File: css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    <main>
+      <div class="container">
+        <div class="row">
+          <div class="column-full">
+            <h1>Choose your Character!</h1>
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-two-third">
+            <div class="column-one-second"><img src="images/link.png" /></div>
```
Comment (shawnkost) [inline css-layout-classes/index.html]: This `column-one-second` class is unnecessary, and causing your images not to display propertly.

The `img` should just be in the `column-two-third` directly

File: css-layout-classes/index.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+              <h2>Super Smash Bros. Ultimate</h2>
+              <p>
+                Super Smash Bros. Ultimate received universal acclaim from
+                both......
+              </p>
+            </div>
+          </div>
+          <div class="column-one-third">
+            <img src="images/yoshi.png" />
+          </div>
```
Comment (shawnkost) [inline css-layout-classes/index.html]: All 3 of these columns should be the same. They should all be a `column-one-third`

File: css-layout-classes/index.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            <div>
+              <p>
+                Donkey Kong is a serious of video games featuring the adventures
+                of a gorilla called Donkey Kong......
+              </p>
+            </div>
+          </div>
+          <div class="column-one-second">
+            <img src="images/dk.png" />
+          </div>
```
Comment (shawnkost) [inline css-layout-classes/index.html]: These columns are different sizes. The one with the text should be taking up 2/3rds whereas the image will be the one-third

File: css-layout-classes/index.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+          <div class="column-one-second">
+            <div><h2>Mario</h2></div>
+            <div>
+              <p>
+                Mario is a fictional character in the Mario video game
+                franchise, owned by Nintendo and created by Japanese......
+              </p>
+            </div>
+          </div>
```
Comment (shawnkost) [inline css-layout-classes/index.html]: These columns are different sizes. The one with the text should be taking up 2/3rds whereas the image will be the one-third

File: css-layout-classes/styles.css [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+.row {
+  display: flex;
+  flex-wrap: nowrap;
+}
+
+.row .column-two-third {
+  display: flex;
+}
+
```
Comment (shawnkost) [inline css-layout-classes/styles.css:31]: Missing selectors for the rest of your columns. Every single one of your `column` classes should have a selector with a `flex-basis` or `width` applied

**Review by shawnkost — CHANGES_REQUESTED** [review]
Few changes here William! Let me know if you have any questions

**Comment by williamzhang73** [comment]
lay

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**